### ISSUE TITLE
Path to Rec

### ISSUE THREAD

**Issue by rob-metalinkage** [openprov/prov-jsonld#2]
About a year ago I created a schema using similar principles, and tested it as an composable element in other schemas (especially the JSON-LD composition is non-trivial)

https://ogcincubator.github.io/bblock-prov-schema/

We have had similar goals, it would be helpful to have a common solution.  It would be good to compare and contract features of each approach and see what an optimum way forward would look like.

The OGC and W3C have a joint WG, so we could potentially progress this topic to a REC status with some testing and implementation.

We have an ongoing activity looking at DCAT profiles - GeoDCAT but can test further for PROV - and have postulated a PROV profile for geospatial might be relevant,  if it is a JSON schema that can be combined with OGC OpenAPI schemas.

A work-in-progress is here: https://ogcincubator.github.io/geodcat-ogcapi-records/bblock/ogc.geo.geodcat.geodcat-records-prov

Reach out to me if anyone is interested in taking this further.

**Comment by lucmoreau** [comment]
Hi Rob,

Thanks for reaching out.
I just wanted to double check you had come across https://www.w3.org/submissions/prov-jsonld/

Kind regards,
Luc

--
Professor Luc Moreau,  Executive Dean
Faculty of Science, Engineering and Medicine | University of Sussex
John Maynard Smith Building, Room 2B3
EA: Janhavi Sharma ***@***.***>







From: Rob ***@***.***>
Date: Tuesday, 10 December 2024 at 06:12
To: openprov/prov-jsonld ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [openprov/prov-jsonld] Path to Rec (Issue #2)

About a year ago I created a schema using similar principles, and tested it as an composable element in other schemas (especially the JSON-LD composition is non-trivial)

https://ogcincubator.github.io/bblock-prov-schema/<https://ogcincubator.github.io/bblock-prov-schema/>

We have had similar goals, it would be helpful to have a common solution. It would be good to compare and contract features of each approach and see what an optimum way forward would look like.

The OGC and W3C have a joint WG, so we could potentially progress this topic to a REC status with some testing and implementation.

We have an ongoing activity looking at DCAT profiles - GeoDCAT but can test further for PROV - and have postulated a PROV profile for geospatial might be relevant, if it is a JSON schema that can be combined with OGC OpenAPI schemas.

A work-in-progress is here: https://ogcincubator.github.io/geodcat-ogcapi-records/bblock/ogc.geo.geodcat.geodcat-records-prov<https://ogcincubator.github.io/geodcat-ogcapi-records/bblock/ogc.geo.geodcat.geodcat-records-prov>

Reach out to me if anyone is interested in taking this further.

**Comment by rob-metalinkage** [comment]
Yes, this is how I navigated to the repo, and added the first issue :-)
